README.md

```diff
@@ -94,7 +94,7 @@ Next, open a terminal window, and enter the following commands:
     $ cd <your_ghostwriter_folder_location>
     $ qmake
     $ make
-    $ make install
+    # make install
```

Comment:
These prompt symbols are kinda undefined, and this makes it look like a comment. Maybe prefixing with `sudo` would make it more obvious?

Reply:
Not all systems have sudo by default.  For example, Debian.  It's usually implied that installing will need admin privileges.